### ISSUE TITLE
Paths_sdl2 module missing

### ISSUE THREAD

**Issue by MEngher** [haskell-game/sdl2#143]

**Comment by ocharles** [comment]
It sounds like you aren't using `cabal`, but you haven't provided enough information. Please compile the examples with `cabal` (if that's what you're trying to build) and re-open if there is still a problem.

**Comment by MEngher** [comment]
Hi, sorry to bother you. I did follow your instructions: download the zip, cabal build. Cabal build did work (all the modules have been compiled, I can see all the .o files). But how do I run one of the lessons? (I am just starting to learn Haskell and Cabal and did not find an explanation on how to use the flags).

Best Regards

Marc

Sent from my iPad

On May 30, 2017, at 09:27, Oliver Charles <notifications@github.com<mailto:notifications@github.com>> wrote:


It sounds like you aren't using cabal, but you haven't provided enough information. Please compile the examples with cabal (if that's what you're trying to build) and re-open if there is still a problem.

**Comment by ocharles** [comment]
`cabal run` *name of example*. You can see all the names in the `.cabal` file (see the `executable` lines)

**Comment by MEngher** [comment]
Thank you. I finally make it run (all examples). However, don’t know why but before installing your package, I did install SDL2 following “what internet says about it”: that is an adventure by itself). I did make a test using lazyfoo-lesson01 and it worked. But after installing  your package (cabal configure, cabal build) I have to reinstall sdl2 and sdl2-image two times. And then it finally worked.

Thank you for your help

Best Regards

Marc
